src/libs/getPlaidDesktopMessage/types.ts

```diff
@@ -0,0 +1,3 @@
+type GetPlaidDesktopMessage = () => string | void;
```

Comment:
```suggestion
type GetPlaidDesktopMessage = () => string | undefined;
```

Based on comment above.

Reply:
Good idea, should be ok now @fabioh8010